extension/cgroupruntimeextension/testdata/config.yaml

```diff
@@ -0,0 +1,7 @@
+cgroupruntime:
+cgroupruntime/invalid_ratio:
+  gomemlimit:
+    ratio: 1.1
+cgroupruntime/invalid_ratio_type:
+  gomemlimit:
+    ratio: "not_valid"
```

Comment:
This will only test that the type in the config is correct but the error will actually come from `confmap`, not from this extension. Maybe it's more interesting to test for negative values?

Reply:
Absolutely, extended the tests cases with a negative and a zero value: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35472/commits/6abcf495a3d6d457cfb9670f1d459fdcdf815fad